esphomeRemote/esphomeRemoteLight.h

```diff
@@ -2,30 +2,172 @@
 #include "DisplayUpdateInterface.h"
 #include "MenuTitle.h"
 #include "FriendlyNameEntity.h"
+#include <sstream>
 
 #pragma once
 
+#define MAX_BRIGHTNESS 255.0f
+/* Home Assistant entities can have these modes in color_mode/supported_color_mode.
+ * Depending on the mode the light supports we want to show sliders or not.
+ *
+ * |color_mode | supports brightness | supports color_temp |
+ * |---------- | ------------------- | ------------------- |
+ * | UNKNOWN   |          -          |          -          |
+ * | ONOFF     |          -          |          -          |
+ * | BRIGHTNESS|          x          |          -          |
+ * | COLOR_TEMP|          x          |          x          |
+ * | HS        |          x          |          -          |
+ * | RGB       |          x          |          -          |
+ * | RGBW      |          x          |          -          |
+ * | RGBWW     |          x          |          x          |
+ * | WHITE     |          x          |          -          |
+ * | XY        |          x          |          -          |

```

Comment:
we could make this into an enum and then have two functions like supportsBrightness and supportsColortemp. then we're just storing the enum value instead of the colormode as a string

Reply:
i think we can remove the comment now because the code is pretty clear

Reply:
done